src/main/java/org/broadinstitute/consent/http/service/DatasetRegistrationService.java

```diff
@@ -106,10 +106,14 @@ private DatasetServiceDAO.DatasetInsert createDatasetInsert(DatasetRegistrationS
                                                                 Integer consentGroupIdx) throws IOException {
         ConsentGroup consentGroup = registration.getConsentGroups().get(consentGroupIdx);
 
-        if (Objects.isNull(dacDAO.findById(consentGroup.getDataAccessCommitteeId()))) {
-            throw new NotFoundException("Could not find DAC");
+        if (Objects.nonNull(consentGroup.getDataAccessCommitteeId())) {
+            if (Objects.isNull(dacDAO.findById(consentGroup.getDataAccessCommitteeId()))) {
+                throw new NotFoundException("Could not find DAC");
+            }
         }
```

Comment:
```suggestion
        if (Objects.nonNull(consentGroup.getDataAccessCommitteeId())
            && Objects.isNull(dacDAO.findById(consentGroup.getDataAccessCommitteeId()))) {
                throw new NotFoundException("Could not find DAC");
        }
```

Would it be worth doing this instead?